flask_cors/extension.py

```diff
@@ -194,7 +194,7 @@ def cors_after_request(resp):
         for res_regex, res_options in resources:
             if try_match(normalized_path, res_regex):
                 LOG.debug("Request to '%s' matches CORS resource '%s'. Using options: %s",
-                      request.path, get_regexp_pattern(res_regex), res_options)
+                      repr(request.path), get_regexp_pattern(res_regex), res_options)
```

Comment:
Note that you could use `%r` in the log message instead, which has the added benefit that `repr()` won't even be called if the log message is not to be emitted (typically because the it's set higher than `DEBUG`).

```suggestion
                LOG.debug("Request to '%r' matches CORS resource '%s'. Using options: %s",
                      request.path, get_regexp_pattern(res_regex), res_options)
```

Reply:
Great suggestion @noirbee!
I've implemented the changes.

Reply:
Sorry for not commiting the suggestion instead of implementing it on my own.